pommerman/agents/player_agent.py

```diff
@@ -74,24 +74,24 @@ def __init__(self, character=characters.Bomber, agent_control='arrows'):
         # and prevents Pommerman from running.
         #
         from pyglet.window import key
-         controls = {
-            'arrows': {
-                key.UP: 1,
-                key.DOWN: 2,
-                key.LEFT: 3,
-                key.RIGHT: 4,
-                key.SPACE: 5,
-                key.M: 6  # In Pommerman, this will freeze the game.
-            },
-            'wasd': {
-                key.W: 1,
-                key.S: 2,
-                key.A: 3,
-                key.D: 4,
-                key.E: 5,
-                key.Q: 6  # In Pommerman, this will freeze the game.
+            controls = {
```

Comment:
I think this should be indented back to line up with `from pylget...`

Reply:
oh actually, was it already in line with it and the `-` is causing it to not appear that way?